less/plugins/adapt-contrib-boxmenu/boxMenu.less

```diff
@@ -65,6 +65,8 @@
   }
 
   &__item-container {
+    margin: 0 -@menu-item-padding-hoz;
```

Comment:
The negative margin is causing a vertical scrollbar to display. Testing on Mac Safari, FF, Chrome and iPhone.

Reply:
Well, that's inconvenient. The nag margins is a bit of an old skool way of doing things now we have gap and flex. It really should be in an additional div that wraps the items. So, I think we have three options for the short term, 1. leave with a smaller margin on the item container / 2. try to fix with additional div etc. or 3. revert

Adjusting a margin should be really easy, but I in some ways at least it highlights what needs addressing . I'm happy to go all 3 routes though.

Reply:
I'd be in favour of option 3 (revert) and instead invest effort looking into improving the column/grid layout as a separate task. However, if people want to see a quick fix for this then I'm happy with that too.

Reply:
also happy either way, @guywillis ?

Reply:
Am in agreement with Kirsty here. The styles need fixing but it's probably best to invest the time into a long term solution.

Reply:
So revert or live with the slight inconsistency ? @guywillis / @kirsty-hames

Reply:
Sorry, definitely remove the neg margins. I meant reverting the margin increase so it was consistent (so for the page also). But can just revert for menu.. Or I was also suggesting we could leave the menu header, but just have the inconsistency in the item container.

Reply:
@kirsty-hames / @guywillis I've reverted the negative margins on the menu items. This does mean that everything apart from the margin around the items is 1rem and here its 0.5 but I've created a new issue to address this here #447